testing/run-scripts/gen_browser.sh

```diff
@@ -16,6 +16,13 @@
 #        - beta
 #        - release
 
+if [ ! -x "${BASH_SOURCE%/*}/utils.sh" ]
+then
+    echo "Script called incorrectly, or improperly installed."
+    exit 1
+fi
+
+source "${BASH_SOURCE%/*}/utils.sh"
```

Comment:
Will the script fail if `utils.sh` can't be found? Maybe this would work:
```bash
source "${BASE_SOURCE%/*}/utils.sh" || echo "cannot find utils.sh" && exit 1
```

(just because it's several lines shorter and repeated elsewhere)